python/ray/scripts/scripts.py

```diff
@@ -759,90 +772,71 @@ def start(
-                # of the cluster. Please be careful when updating this line.
-                cli_logger.print(
-                    cf.bold("  ray start --address='{}'"),
-                    bootstrap_address,
-                )
-                cli_logger.newline()
-            if ray_constants.ENABLE_RAY_CLUSTER:
+
+            cli_logger.newline()
+            cli_logger.print(
+                "To connect to this Ray cluster, run `ray.init()` as usual:"
+            )
+            with cli_logger.indented():
+                cli_logger.print("{} ray", cf.magenta("import"))
                 cli_logger.print(
-                    "To connect to this Ray cluster, run `ray.init()` as usual:"
-                )
-                with cli_logger.indented():
-                    cli_logger.print("{} ray", cf.magenta("import"))
-                    cli_logger.print(
-                        "ray{}init()",
-                        cf.magenta("."),
+                    "ray{}init({})",
+                    cf.magenta("."),
+                    "_node_ip_address{}{}".format(
+                        cf.magenta("="), cf.yellow("'" + node_ip_address + "'")
                     )
-                cli_logger.newline()
-                cli_logger.print(
-                    "To connect to this Ray instance from outside of "
-                    "the cluster, for example "
+                    if include_node_ip_address
+                    else "",
                 )
+            cli_logger.newline()
+            cli_logger.print(
+                "To connect to this Ray instance from outside of "
+                "the cluster, for example "
+            )
+            cli_logger.print(
+                "when connecting to a remote cluster from your laptop, " "make sure the"
+            )
+            cli_logger.print(
+                "dashboard {}is accessible and use the Ray Jobs API. For example:",
+                f"({dashboard_url}) " if dashboard_url else "",
+            )
+            if dashboard_url:
                 cli_logger.print(
-                    "when connecting to a remote cluster from your laptop, "
-                    "make sure the"
+                    cf.bold(
+                        "  RAY_ADDRESS='http://<dashboard URL>:{}' ray job submit "
+                        "--working-dir . "
+                        "-- python my_script.py"
+                    ),
+                    ray_params.dashboard_port,
                 )
+            cli_logger.newline()
+            cli_logger.print(
+                "See https://docs.ray.io/en/latest/cluster/running-applications"
+                "/job-submission/index.html"
+            )
+            cli_logger.print(
+                "for more information on connecting to the Ray cluster from "
+                "a remote client."
+            )
+            cli_logger.newline()
+            cli_logger.print("To see the status of the cluster, use")
```

Comment:
```suggestion
            cli_logger.print("To view the status of the cluster, use")
```

Reply:
Is view better than see?

Reply:
I think so. I don't find "see" used very often. It seems @angelinalg also suggested this kind of change here: https://github.com/ray-project/ray/pull/32854#discussion_r1145534482.